### ISSUE TITLE
Smoother Integration With Google Drive

### ISSUE THREAD

**Issue by pelletier197** [neo4j-labs/arrows.app#55]
Is there any reason why this application is not registered on the Google Workspace Marketplace? https://workspace.google.com/marketplace

The way the application works right now is convenient when there is only one person using/editing the graph model, but is not really suited for developing with multiple people than can access/modify the file. Every person will need to connect to the application and grant access, which becomes redundant. 

Integrating with Google Workspace Marketplace would allow an organization/company to add the application to their workspace and all members of the workspace could use it as if it was any other application (like google slides or whatever). 

All and all, really great application, thank you!

**Comment by apcj** [comment]
Hi @pelletier197 thanks for the feedback, and will definitely consider registering as you suggest in the future.

I totally agree that for now the app only works well when one person edits a graph at a time. Simultaneous editing can cause very undesirable behaviour. We have some ideas in the works for for how to improve this significantly, but there will still be a long way to go to achieve the kind of real-time collaborative editing that you might be familiar with from the core Google apps.

**Comment by pelletier197** [comment]
Thank you for your reply! I totally understand, and it's perfectly fine. Right now, we're using draw.io where I am working to document the graph, and I can say that I would much rather use Arrows instead If could. It's much more straightforward with this application. 

To be fair, I don't think it would be an issue right now for us if simultaneous edit was not supported. We don't modify that model very often. But clearly, having real time multi-edit like Google apps would be the greatest of all.

**Comment by apcj** [comment]
Hi again @pelletier197 thanks for the insight. I'd love to understand more about the hurdles with the current behaviour. When a new-to-arrows user wants to edit an arrows file for the fist time, it should be sufficient for them to click on a link to arrows.app (including the Google document id) and the page should ask for their permission straight away. Is this the step that you want to eliminate, or is there some other inconvenience that I'm not thinking of?

**Comment by pelletier197** [comment]
Yeah pretty much. I believe that when you have an app in a Google workspace, you give permissions to the app inside the workspace, and then app should have access without the need for everyone to grant permission.

And I also believe that some organization may block their users from installing applications on their own workspace for security reasons, so new applications have to be approved and installed from the Marketplace.

Although, I'll be honest, I'm not enterly sure of all I've said. And maybe registering in the Marketplace wouldn't change anything except making the application easier to install and more trustable.

**Comment by apcj** [comment]
Thanks @pelletier197 that does make sense. I am also not entirely sure of the implications, so will do some more research before picking this up.
